conbench/entities/hardware.py

```diff
@@ -188,9 +188,27 @@ class MachineCreate(marshmallow.Schema):
 
 
 class ClusterCreate(marshmallow.Schema):
-    name = marshmallow.fields.String(required=True)
-    info = marshmallow.fields.Dict(required=True)
-    optional_info = marshmallow.fields.Dict(required=True)
+    name = marshmallow.fields.String(
+        required=True,
+        metadata={
+            "description": "Information related to cluster (e.g. `hosts`, `nodes` or `number of workers`)"
+            "configured to run a set of benchmarks. Used to differentiate between similar"
+            "benchmark runs performed on different sets of hardware"
+        },
+    )
+    info = marshmallow.fields.Dict(
+        required=True,
+        metadata={
+            "description": "`Distinct` name of the cluster, to be displayed on the web UI."
+        },
+    )
+    optional_info = marshmallow.fields.Dict(
+        required=True,
+        metadata={
+            "description": "Additional optional information about the cluster, which may not impact the"
```

Comment:
A very minor style edit (though feel free to disregard if you think it makes this overly complicated) 

```suggestion
            "description": "Additional optional information about the cluster, which is likely to not impact the"
```

Reply:
`likely` suggests a tone similar to `probably`, which sounds great.